CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+
+## [Unreleased]
+### Added
+  * [Behat/Behat#976](https://github.com/Behat/Transliterator/pull/18): Stop \Behat\Transliterator\Transliterator::postProcessText() breaking words containing apostrophes
```

Comment:
I've just added in something more like the rest of this library's changelog, will push shortly.